web-client/src/app/views/send-funds/send-funds.page.ts

```diff
@@ -12,15 +13,32 @@ export class SendFundsPage implements OnInit {
       label: 'Quick pay',
       title: 'Scan a QR code',
       icon: faQrcode,
+      action: 'presentScanner',
     },
     {
       label: 'Add New Friend',
       title: 'Share my wallet address',
       icon: faLink,
+      disabled: true,
     },
   ];
 
-  constructor() {}
+  constructor(private scannerService: ScannerService) {}
 
   ngOnInit() {}
+
+  async presentScanner() {
+    await this.scannerService.presentScanner();
+  }
+
+  perform(action: string | undefined) {
```

Comment:
Maybe rename this function to "perform_action" or something that describes what the function does in a clearer way.